### ISSUE TITLE
Experimental settings

### ISSUE THREAD

**Issue by yaorz97** [Wuziyi616/multi_part_assembly#9]
I have some questions about the experimental settings:
1) In subsection 6.1, you said that models were separately trained on fractured objects with at most 20 fractured pieces for each category. I find some categories only contain very few models, such as Cookie (4 * 100 models), and Statue (4* 100 models), while some categories have more models, such as Vase (106 *100) and Bottle (73 *100). Did you use some methods to avoid overfitting when training on the categories containing few models? 
2)  The experimental results in the top block of Table 5 were trained and tested on the artifact subset? The experimental results in the bottle block of Table 5 were obtained by fine-tuning from the models trained on the everyday subset? There were 20 models trained on each category, so which one did you use to do fine-tuning?

**Comment by Wuziyi616** [comment]
1. This is a good question and something to study. In our benchmark, I use **the same training setting** for all models on all categories (e.g. same learning rate, same epochs). And yes, I did observe overfitting on categories with a small number of data
2. Yes, the top one is training from scratch and the bottom one is doing finetuning, where you can see finetuning (transfer learning) leads to better results than training from scratch. We not only train 20 models for 20 categories, but we also have results where we train **a single model** over all 20 categories (see [paper](https://arxiv.org/pdf/2210.11463.pdf) Table 11 in the last page). So we use this model trained on all categories for transfer learning.

**Comment by yaorz97** [comment]
Thank you for your prompt response!

**Comment by yaorz97** [comment]
What's more, I want to know if the model was trained in all categories in the 6.2 ablation study, or 20 models were trained with respect to each category.

**Comment by Wuziyi616** [comment]
I think that one is trained on each category **separately**. Actually I believe all the models in the main paper is trained on each category separately. We only put joint trained results in the appendix
